server/graphql/v2/object/OrderPermissions.ts

```diff
@@ -30,6 +30,15 @@ export const canSeeOrderPrivateActivities = async (req: express.Request, order):
   return isHostAdmin(req, order);
 };
 
+export const canSetOrderTags = async (req: express.Request, order): Promise<boolean> => {
```

Comment:
For consistency, I would suggest renaming this to `canEditOrderTags`.

Reply:
The mutation itself is named `setTags`, `canSetTags` seems consistent on this regard